docs/changes/2636.features.rst

```diff
@@ -0,0 +1 @@
+Makes changes to the CameraCalibrator in ctapipe.calib.camera.calibrator that allows it to correctly variance images generated with the VarianceExtractor
```

Comment:
Please describe the actual change, not just say "there are changes". Also there is a verb missing here (calibrate?)

Reply:
i added some explanation